src/components/Modals/TaskStatusUpdate/index.tsx

```diff
@@ -47,7 +47,6 @@ export default function TaskStatusUpdate() {
 
   return (
     <TaskStatusUpdatePresentation
-      isOpen={isTaskNoteworthy}
```

Comment:
we have the condition in layout as well for rendering the component, so it doesn't make sense to have this this prop here

NOTE: the state used here was incorrect